### PR TITLE
Feature/refactoring branch dau

### PR DESCRIPTION
**Pull request by SHGODE**
Opening a PR where flake 8 issues are resolved.

### CODE REVIEW

File: app/core/tasks.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from core.management.commands.validate_source_metadata import \
+    Command as validate_source_Command
+from core.management.commands.validate_target_metadata import \
+    Command as validate_target_Command
+
+logger = logging.getLogger('dict_config_logger')
+
+
+@shared_task(name="workflow_for_xia")
+def xia_workflow():
```
Comment (mattdrayer) [inline app/core/tasks.py:22]: Let's go with a verb/action-oriented name for this function: `xia_workflow` -> `execute_xia_workflow`, `execute_xia_automated_workflow`, `execute_full_xia_worklfow`, etc.

File: docker-compose.yml [hunk 2]
```diff
@@ -43,6 +45,52 @@ services:
...
+    networks:
+      - openlxp
+    restart: on-failure
+
 networks:
   openlxp:
     external: true
+
+volumes:
+  mysql_data:
```
Comment (mattdrayer) [inline docker-compose.yml:96]: Looks like we are missing a newline here

File: requirements.txt [hunk 3]
```diff
@@ -2,7 +2,7 @@ mysqlclient>=2.0.3,<2.1.0
 
 django-mysql>=3.10.0,<3.11.0
 
-Django>=3.1.4,<3.2.0
+Django>=3.1.6,<3.2.0
```
Comment (mattdrayer) [inline requirements.txt:5]: Is this an across-the-board update, or just something that is specific to this one XIA DAU component?

**Review by mattdrayer — APPROVED** [review]
Just a few small comments -- generally speaking the changes LGTM.

File: requirements.txt (hunk 3 above)
Comment (SHGODE) [reply requirements.txt:5]: trying to keep this across all agents.

**Review by KarenAJ — APPROVED** [review]
Approved!

**Pull request merged**